src/test/java/gumtree/spoon/AstComparatorTest.java

```diff
@@ -1102,21 +1131,30 @@ public void test_t_225262() throws Exception {
 	}
 
 	@Test
+	@Ignore
```

Comment:
I am assuming this because of the move operation not being detected. Correct?

Reply:
The edit script generated was quite complex. I ignore it to avoid blocking the bump

Reply:
Okay. Better to state this in the code itself.
```suggestion
	@Ignore("Edit script generated is too complex. Ignoring in order to upgrade to GT3")
```